schema/ont_papers.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix : <http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/> .
```

Comment:
Wrong empty prefix.

Reply:
Sorry it was added automatically and I overlooked it, I will fix it